tcav/imagenet_and_broden_fetcher.py

```diff
@@ -0,0 +1,268 @@
+import pandas as pd
+import urllib.request
+import os
+import shutil
+import PIL
+from PIL import Image
+import tensorflow as tf
+import socket
+import random
+
```

Comment:
include top-level comment about what this file is used for and how to call it

Reply:
Done